### PR TITLE
Created Products Page

### PR DESCRIPTION
**Pull request by khyatibaria**
### Created Products Page
Please review https://deploy-preview-5--bookish-club.netlify.app/products

### CODE REVIEW

File: products.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+body {
+  background-color: #f5f5f5;
```
Comment (bharati-21) [inline products.css:2]: You can maintain consistency by using either variables or color hex/ names.

File: products.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  background-color: white;
+}
+
+.card {
+  background-color: #f5f5f5;
```
Comment (bharati-21) [inline products.css:38]: Similar suggestion.

**Review by NeerajVodala — APPROVED** [review]
Try to declare colors as variables so that it is easy to use them. Rest everything looks good to me.
All the best.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged.

**Pull request merged**